libs/clients/vmst/src/lib/utils.ts

```diff
@@ -8,39 +56,82 @@ export const createWrappedFetchWithLogging = (
   return new Promise((resolve, reject) => {
     fetch(input, init)
       .then(async (response) => {
-        const body = init?.body ? JSON.parse(init?.body as string) : {}
+        let requestBody = init?.body ? JSON.parse(init?.body as string) : {}
 
-        const newInit = {
-          ...init,
-          body: {
-            ...body,
-            applicant: 'hidden',
-            otherParentId: '' ?? 'hidden',
-            email: 'hidden',
-            phoneNumber: 'hidden',
-            paymentInfo: {
-              ...body?.paymentInfo,
-              bankAccount: 'hidden',
-            },
-            attachments: body?.attachments?.length,
+        // Filter known sensitive data
+        // First pick what we need
+        requestBody = pick(requestBody, [
+          'adoptionDate',
+          // 'applicant',
+          'applicationComment',
+          'applicationFundId',
+          'applicationId',
+          'attachments',
+          'dateOfBirth',
+          // 'email',
+          'employers',
+          'expectedDateOfBirth',
+          'otherParentBlocked',
+          // 'otherParentId',
+          'paymentInfo',
+          'periods',
+          // 'phoneNumber',
+          'rightsCode',
+          'status',
+          // 'testData',
```

Comment:
```suggestion
          'testData',
```

Reply:
Why do you want `testData`?

Reply:
if applicant has employer/other parent this api is called as a test application to see if there are any errors when sending in the application (testData = true), then testData = false is when actually sending in the application to vmst